Buildings/Fluid/Humidifiers/EvaporativeCoolers/IndirectDry.mo

```diff
@@ -0,0 +1,141 @@
+within Buildings.Fluid.Humidifiers.EvaporativeCoolers;
+model IndirectDry "Indirect dry evaporative cooler"
+
+  extends Buildings.Fluid.Interfaces.PartialFourPortParallel(
+    redeclare final package Medium1=MediumPri,
+    redeclare final package Medium2=MediumSec);
+
+  replaceable package MediumPri =
+    Modelica.Media.Interfaces.PartialMedium
+    "Medium 1 in the component"
+      annotation (choices(
+        choice(redeclare package Medium = Buildings.Media.Air "Moist air"),
+        choice(redeclare package Medium = Buildings.Media.Water "Water"),
+        choice(redeclare package Medium =
+            Buildings.Media.Antifreeze.PropyleneGlycolWater (
+          property_T=293.15,
+          X_a=0.40)
+          "Propylene glycol water, 40% mass fraction")));
+
+  replaceable package MediumSec =
+    Modelica.Media.Interfaces.PartialMedium
+    "Medium 2 in the component"
+    annotation (choices(
+        choice(redeclare package Medium = Buildings.Media.Air "Moist air")));
+
+  parameter Real eps(
+    final unit = "1")=0.8
+    "Heat exchanger effectiveness";
+
+  parameter Real effCoe[11]={ 0.792714, 0.958569, -0.25193, -1.03215, 0.0262659,
+                              0.914869, -1.48241, -0.018992, 1.13137, 0.0327622,
+                              -0.145384}
+    "Coefficients for evaporative medium efficiency calculation";
+
+  parameter Modelica.Units.SI.Pressure dp_nom
+    "Nominal pressure drop";
```

Comment:
Do you mean `dp_nominal`? Using different terms than what is common is confusing to the user.